server/src/main/java/io/crate/metadata/sys/SysShardsTableInfo.java

```diff
@@ -141,6 +142,14 @@
                 .add("stage", STRING, ShardRowContext::recoveryStage)
                 .add("type", STRING, ShardRowContext::recoveryType)
                 .add("total_time", LONG, ShardRowContext::recoveryTotalTime)
+                .startObject("source_node")
```

Comment:
Do we want to expose these by now? If not, some tests needed to be adjusted to use ES API.
If yes, I'd add relevant docs and changes entries.